src/types/student/student.enum.ts

```diff
@@ -0,0 +1,16 @@
+export enum TypeWork {
+  'Praca w biurze' = 1,
+  'Praca zdalna' = 2,
+}
+
+export enum ContractType {
+  'Umowa o pracę' = 1,
+  'B2B' = 2,
+  'Umowa zlecenie' = 3,
+  'Umowa o dzieło' = 4,
+}
+
+export enum Internship {
+  'Nie',
```

Comment:
A tu nagle brak. Poza tym po co w `'Nie'`, wystarczy `Nie`. Już pomijam, że lepiej jakby to było po angielsku raczej :)